### ISSUE TITLE
fx2lafw_eeprom_loader build issue on linux

### ISSUE THREAD

**Issue by iDoka** [podonoghue/LogicAnalyser#2]
I try to build fx2lafw_eeprom_loader on CentOS 7.8, but it fail.
Pre-builded version also fail:
```
$ ./fx2lafw_eeprom_loader
./fx2lafw_eeprom_loader: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.20' not found (required by ./fx2lafw_eeprom_loader)
./fx2lafw_eeprom_loader: /lib64/libstdc++.so.6: version `CXXABI_1.3.9' not found (required by ./fx2lafw_eeprom_loader)
./fx2lafw_eeprom_loader: /lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by ./fx2lafw_eeprom_loader)
```

===================================

## Step to reproduce:
```
git clone https://github.com/podonoghue/LogicAnalyser
cd LogicAnalyser/Software/fx2lafw_eeprom_loader/src
g++ fx2lafw_eeprom_loader.cpp  -lusb -L/usr/lib64 -std=c++11 -m64 -fPIC -g -O2 -o fx2lafw_eeprom_loader

/usr/bin/ld: /tmp/ccrMroFR.o: undefined reference to symbol 'libusb_get_device_descriptor'
/usr/lib64/libusb-1.0.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

Interesting things related to `libusb_get_device_descriptor`:
```
$ grep -n libusb_get_device_descriptor LibusbCpp.h
198:         int rc = libusb_get_device_descriptor(currentDevice, &deviceDescriptor);

$ grep -n libusb_get_device_descriptor libusb.h
1321:int LIBUSB_CALL libusb_get_device_descriptor(libusb_device *dev,
1903:	 * It is safe to call libusb_get_device_descriptor on a device that has left */
```

but mention in libusb.so absent:
```
$ readelf -s /usr/lib64/libusb-0.1.so.4.4.4 | grep libusb_get_device_des
    32: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND libusb_get_device_descrip

$ readelf -s /usr/lib64/libusb-1.0.so.0.1.0 | grep libusb_get_device_des
   101: 0000000000007660    72 FUNC    GLOBAL DEFAULT   11 libusb_get_device_descrip
```

Which version of libusb I should use to get `libusb_get_device_descriptor`?

**Comment by tlambertz** [comment]
I just had a very similar issue, solved it by linking against `libusb-1.0`:

`g++ fx2lafw_eeprom_loader.cpp  -lusb-1.0 -L/usr/lib64 -std=c++11 -m64 -fPIC -g -O2 -o fx2lafw_eeprom_loader`

**Comment by iDoka** [comment]
@tlambertz thanks for your comment.
It works for me
